From a35927899af8c68b36b3ad8876fd6aecabce7001 Mon Sep 17 00:00:00 2001
From: David Zeuthen <davidz@redhat.com>
Date: Sun, 13 Sep 2009 17:55:34 +0000
Subject: Add missing comma so we're save both LANG and LANGUAGE, not only LANGLANGUAGE

---
diff --git a/src/programs/pkexec.c b/src/programs/pkexec.c
index 89b958e..4eeb55f 100644
--- a/src/programs/pkexec.c
+++ b/src/programs/pkexec.c
@@ -174,7 +174,7 @@ main (int argc, char *argv[])
   gchar pwbuf[8192];
   gchar *s;
   const gchar *environment_variables_to_save[] = {
-    "LANG"
+    "LANG",
     "LANGUAGE",
     "LC_ALL",
     "LC_MESSAGES",
--
cgit v0.8.2
